schemas/manifest.fbs

```diff
@@ -5,21 +5,30 @@ table Uuid {
     low: uint64;
 }
 
-table DbParent {
-    // Path to the parent database from which this database was cloned
+// Reference to an external database.
+table ExternalDb {
+    // Path to root of the external database
     path: string (required);
 
-    // Parent checkpoint ID that this clone was derived from
-    checkpoint: Uuid (required);
+    // Externally owned Checkpoint ID we've used to create an initial state of cloned database.
+    source_checkpoint_id: Uuid (required);
+
+    // Checkpoint ID this database has placed on the external database that prevents referenced
+    // data files from being GC'd. Both final_checkpoint_id and source_checkpoint_id should resolve
+    // to the same manifest_id as long as they both still exist.
+    final_checkpoint_id: Uuid;
+
+    // Compacted SST IDs belonging to external DB that are currently being referenced.
+    sst_ids: [CompactedSstId] (required);
```

Comment:
I'm a little confused why we don't need to store the sorted runs for these SST IDs. I read over all the code and didn't see any changes related to SR's. How is the cloned manifest dealing with SRs for SST IDs from external DBs?

Reply:
> when we scan through the SortedRun

Ah, I see. I missed the part where we're inheriting SRs from parent manifests as we clone, including transitively/deeply nested clones. Gotcha.